### ISSUE TITLE
jsonpath 2.2.0 - Invalid OSGi involving the inclusion of org.apache.tapestry5.json

### ISSUE THREAD

**Issue by iragreenstein** [jayway/JsonPath#227]
Greetings,

My apologies if i should open a new issue for this.

I am making use of version 2.2.0, however sans camel. Using the maven dependency found in http://mvnrepository.com/artifact/com.jayway.jsonpath/json-path/2.2.0, i too am seeing that tapestry is not optional:

From the MANIFEST.MF

```
Manifest-Version: 1.0
Implementation-Title: json-path
Bundle-SymbolicName: com.jayway.jsonpath.json-path
Implementation-Version: 2.2.0
Bundle-ManifestVersion: 2
Bnd-LastModified: 1456761826000
Import-Package: 
    org.json;resolution:=optional,
    com.google.gson;resolution:=optional;version="[2.3,3)",
    com.google.gson.internal;resolution:=optional;version="[2.3,3)",
    com.google.gson.reflect;resolution:=optional;version="[2.3,3)",
    com.fasterxml.jackson.core;resolution:=optional;version="[2.6,3)",
    com.fasterxml.jackson.databind;resolution:=optional;version="[2.6,3)",
    com.fasterxml.jackson.databind.node;resolution:=optional;version="[2.6,3)",
    com.fasterxml.jackson.databind.type;resolution:=optional;version="[2.6,3)",
    net.minidev.json;resolution:=optional;version="[2.2,3)",
    net.minidev.json.parser;resolution:=optional;version="[2.2,3)",
    net.minidev.json.writer;resolution:=optional;version="[2.2,3)",
    org.apache.tapestry5.json,
    org.slf4j;version="[1.7,2)"
```

This also references [#208](https://github.com/jayway/JsonPath/issues/208), #207[](https://github.com/jayway/JsonPath/issues/207) and [#209](https://github.com/jayway/JsonPath/pull/209)

**Comment by jochenberger** [comment]
This was fixed in #209.

**Comment by kallestenflo** [comment]
Fixed in #209
